### ISSUE TITLE
Wails spams AppData folder

### ISSUE THREAD

**Issue by merlinaudio** [wailsapp/wails#2255]
### Description

In my AppData folder, wails is creating a new folder every time my app... restarts? I'm not exactly sure when it's happening, but it's happening:

![image](https://user-images.githubusercontent.com/85179170/210063733-d565f24c-d840-4417-8db0-471ef9245ccc.png)

All of these contain one folder called EBWebView. This could become problematic because each folder is almost 100 MB in size

![image](https://user-images.githubusercontent.com/85179170/210063962-95cf4443-4dd6-402b-b8c5-fa55d9157e1d.png)

After a short while of using wails, I got around 15 of those folders, so more than a GB of unneeded data

My wails app is in a folder called **`app`**. It appends `-dev` every single time, no idea why

### To Reproduce

Use your app for a while

### Expected behaviour

There would be only one app-dev.exe folder

### System Details

```shell
Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | Windows 10 Pro
Version      | 2009 (Build: 25267)
ID           | 22H2
Go Version   | go1.19.3
Platform     | windows
Architecture | amd64

# Wails

Version | v2.3.1

# Dependencies

Dependency | Package Name | Status    | Version
WebView2   | N/A          | Installed | 108.0.1462.54
npm        | N/A          | Installed | 9.2.0
*upx       | N/A          | Available |
*nsis      | N/A          | Available |
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/
```
```


### Additional context

_No response_

**Comment by merlindru** [comment]
Note that I'm on Windows 11.

Windows 11 really is just a fancy Windows 10 update, so it says "Windows 10 Pro 22H2"

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

That was mainly caused by a bug in `wails dev` which for every reload changed the binary name. Normally the binary name remains the same for every reload and therefore the folder is always the same.

The underlying bug was fixed with #2103 and the fix is available with the latest v2.3.1 release.

**Comment by stffabi** [comment]
Please feel free to reopen if you see any further problems with that.

**Comment by merlindru** [comment]
Thank you!!

**Comment by Osbornnnnn** [comment]
> Thanks for using Wails and reporting this issue.
> 
> That was mainly caused by a bug in `wails dev` which for every reload changed the binary name. Normally the binary name remains the same for every reload and therefore the folder is always the same.
> 
> The underlying bug was fixed with #2103 and the fix is available with the latest v2.3.1 release.

If you change the name of the binary file after `wails build`, it will be the same as described in this problem
![{D941EF88-AA61-4F5F-9FCF-FFC3C9C5A08C}](https://github.com/user-attachments/assets/89dba5db-5ff4-46f0-a9a3-96ac01ad9d1f)
